docs/architecture_en.md

```diff
@@ -116,9 +116,62 @@ At a certain point both are similar because in the end the back-channel logout m
 1. CasIdentityProvider invalidates user's JWT
    - invalidated JWT updates the original JWT in the session store
 
+#### Authentication with Proxy Tickets
+
+Proxy tickets are a mechanism for indirect authentication of users, but without using their original password.
+
+In addition to SonarQube, there may be other services that CAS authenticates to. The CAS specification speaks here of *
+services*. A login via proxy ticket is then no longer done via SonarQube in interaction with CAS, but via another
+service: A proxy service. The person using the ticket must instead authenticate himself in interaction with the proxy
+service. This proxy service is then issued a proxy granting ticket, which in turn can be used to request proxy tickets.
+
+Proxy tickets are very similar to CAS service tickets and have a short validity period. The proxy service makes the
```

Comment:
Here the responsible Java class should be listed in order to get a feeling which class is responsible for what

Reply:
done, also in the german doc translation